File: prelabs/pl6jlassets/snippet2.jl
using DSP
using FFTW 
 # Only use to plot FFT

F  = fft(NoisyData) |> fftshift
freqs = DSP.fftfreq(length(t), 1.0/T) |> fftshift
p = plot(freqs, abs.(F), title = "Spectrum", lw=3, xlim=(0, +250), xlabel="Frequency", ylabel="Amplitude", label="FFT") 

plot!(p, framestyle=:origin, xguide="Time (s)", yguide="A", linecolor=colors, title="", background_color=:transparent, foreground_color=:black, size=(800, 400); grid=true, minorgrid=true) #hide

savefig("snippet2.svg") #hide

File: prelabs/pl6assets/snippet2.m
y = fft(NoisyData);
L = length(t);
P2 = abs(y/L);
P1 = P2(1:L/2+1);
P1(2:end-1) = 2 * P1(2:end-1);
f = Fs*(0:(L/2))/L;
plot(f(1:end/2),P1(1:end/2)); xlabel('Frequency [Hz]'); ylabel('|P1|');